<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>498</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 250, 235);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>425</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>480</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>  Привязка  </string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>661</width>
          <height>101</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch1Time">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>401</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>409.2 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>B1</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>661</width>
          <height>111</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch2Time">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>401</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>354.3 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>B2</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>771</width>
          <height>51</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>751</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>771</width>
          <height>81</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="labelStErr">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>751</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelPulse">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>751</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Bitstream Charter</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <zorder>labelStErr</zorder>
        <zorder>labelPulse</zorder>
        <zorder>labelPulse</zorder>
       </widget>
       <zorder>frame_7</zorder>
       <zorder>frame</zorder>
       <zorder>frame_3</zorder>
       <zorder>frame_6</zorder>
      </widget>
      <widget class="QWidget" name="tab2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <attribute name="title">
        <string>Формирование столов </string>
       </attribute>
       <widget class="QFrame" name="frame_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>781</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch12Time">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>10</y>
           <width>221</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>6100 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>541</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Serif</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Импульс интегратора основного стола</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>781</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch9Time">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>10</y>
           <width>221</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>5000 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>541</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Serif</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>B-импульс промежуточного стола</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>781</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch10Time">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>10</y>
           <width>221</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>5100 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>531</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Serif</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Импульс интегратора промежуточного стола</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>781</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch11Time">
         <property name="geometry">
          <rect>
           <x>550</x>
           <y>10</y>
           <width>221</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>16000 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>541</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Serif</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>B-импульс основного стола</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_26">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>781</width>
          <height>51</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>751</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>  В-таймер</string>
       </attribute>
       <widget class="QFrame" name="frame_18">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>30</y>
          <width>376</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch20Time">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>211</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>1000 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ch4</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_19">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>150</y>
          <width>376</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch21Time">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>211</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>1000 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ch5</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_20">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>376</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch17Time">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>10</y>
           <width>191</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>1000 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ekp</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_22">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>376</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch18Time">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>10</y>
           <width>191</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>1000 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Бампы</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_23">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>376</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch19Time">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>10</y>
           <width>191</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>1000 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ch3</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_24">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>260</y>
          <width>376</width>
          <height>86</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="Ch22Time">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>10</y>
           <width>211</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>1000 Gs</string>
         </property>
        </widget>
        <widget class="QLabel" name="label1_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ch6</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_27">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>781</width>
          <height>51</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>751</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
